docs/content/dagster-cloud/managing-deployments/branch-deployments.mdx

```diff
@@ -63,6 +63,84 @@ Branch Deployments aren't currently supported for use in Dagster Open Source.
+
+#### New assets
+
+Change Tracking will indicate if an asset is new in the Branch Deployment. <Image src="/images/dagster-cloud/managing-deployments/change-tracking-new.png" width={764} height={308} />
+
+#### Code version changes
+
+Change Tracking will indicate if the `code_version` of an asset has changed. Change Tracking can detect added, removed, or updated code versions. For example, if the main branch contains `customers` with code version `v1`
+
+```python file=/dagster_cloud/branch_deployments/change_tracking_code_version.py startafter=start_main_deployment endbefore=end_main_deployment dedent=4
+@asset(code_version="v1")
+def customers(): ...
+```
+
+And the pull request modifies `customers` and changes the code version to `v2`:
+
+```python file=/dagster_cloud/branch_deployments/change_tracking_code_version.py startafter=start_branch_deployment endbefore=end_branch_deployment dedent=4
+@asset(code_version="v2")
+def customers(): ...
+```
+
+The Branch Deployment will mark `customers` as changed. <Image src="/images/dagster-cloud/managing-deployments/change-tracking-code-version.png" width={1042} height={444} />
+
+Some dagster integrations, like `dagster-dbt`, automatically compute code versions for you. For more information on code versions, including strategies for how to compute code versions automatically, see the code versions documentation LINK.
+
+<!-- TODO - question about code versions - are they set to the latest run id if not user specified? docs claim yes but i only see code versions in the ui for dbt assets -->
+
+#### Dependency changes
+
+Change Tracking will indicate if the set of upstream dependencies for an asset has changed. Change Tracking can detect added or removed upstream dependencies. For example, if the main branch contains `returns`:
+
+```python file=/dagster_cloud/branch_deployments/change_tracking_dependencies.py startafter=start_main_deployment endbefore=end_main_deployment dedent=4
+@asset(deps=[orders])
+def returns(): ...
+```
+
+And the pull request adds a new upstream dependency `customers`:
+
+```python file=/dagster_cloud/branch_deployments/change_tracking_dependencies.py startafter=start_branch_deployment endbefore=end_branch_deployment dedent=4
+@asset(deps=[orders, customers])
+def returns(): ...
+```
+
+The Branch Deployment will mark `returns` as having changed dependencies. <Image src="/images/dagster-cloud/managing-deployments/change-tracking-dependencies.png" width={1154} height={490} />
+
+Note that if an asset is marked as having changed dependencies, it means that either an upstream asset has been added or removed as a dependency. It does not mean that the upstream dependency has changed or that the upstream dependency has new data. TODO wording
```

Comment:
1. bold the "not" in "not mean"
2. remove the todo, there's a GH comment for it now